ow_faults_injection/src/FaultInjector.cpp

```diff
@@ -75,33 +75,40 @@ void FaultInjector::jointStateCb(const sensor_msgs::JointStateConstPtr& msg)
   m_joint_state_remapped_pub.publish(output);
 }
 
-void FaultInjector::distPitchFtSensorCb(const geometry_msgs::WrenchStamped& msg)
+void FaultInjector::ArmEndEffectorForceTorqueSensorCb(const geometry_msgs::WrenchStamped& msg)
 {
   if (!m_faults.groups.ft_sensor_faults.enable) {
-    m_dist_pitch_ft_sensor_pub.publish(msg);
+    m_arm_end_effector_force_torque_sensor_pub.publish(msg);
     return;
   }
 
-  auto out_msg = msg;
+  owl_msgs::ArmEndEffectorForceTorque out_msg;
+  out_msg.header = msg.header;
 
   if (m_faults.groups.ft_sensor_faults.zero_signal_failure) {
-    out_msg.wrench.force = geometry_msgs::Vector3();
-    out_msg.wrench.torque = geometry_msgs::Vector3();
+    out_msg.value = {0.0};
```

Comment:
I find this a little bit misleading. When I saw this I thought it was an array fill method that I had never seen before. But in fact what is being done here is explicitly assigning the first value to zero, then zero-initializing all the remaining values. In the end it gets you the result that you want, but I feel it misleads the reader. 
Some alternatives I would propose
Option 1: all 
```suggestion
    out_msg.value = {0.0, 0.0, 0.0, 0.0, 0.0, 0.0};
```
or Option 2: Nothing (haven't tested this but I think it works)
```suggestion
    out_msg.value = {};
```
Option 3: Use a method which would require an additional std include of **algortihm**
```suggestion
std:fill(out_msg.value.begin(), out_msg.value.end(), 0.0);
```

Reply:
I agree that @AstroStucky 's suggestions would be clearer.  I like the first and third best.